core/src/main/scala/kafka/raft/KafkaRaftManager.scala

```diff
@@ -292,12 +260,8 @@ class KafkaRaftManager[T](
     (controllerListenerName, networkClient)
   }
 
-  override def leaderAndEpoch: LeaderAndEpoch = {
-    client.leaderAndEpoch
-  }
-
-  override def voterNode(id: Int, listener: ListenerName): Option[Node] = {
-    client.voterNode(id, listener).toScala
+  override def voterNode(id: Int, listener: ListenerName): Optional[Node] = {
+    client.voterNode(id, listener)
```

Comment:
If the delegation is only a single line and the `client` is an interface, perhaps we don't need to wrap it with `RaftManager`